low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -110,6 +111,41 @@ def _call_script(script_dir: str, args: list):
         script_args_lst = [script_dir]+args
         subprocess.call(script_args_lst, shell=False)
```

Comment:
This will proceed even the script execution fails. We have to capture the stderr and exit for non-zero return code. 
Please confirm.

Reply:
Yes.
Done changes. Please check